File: BarCode SDK/VB.NET/QR Code With Message/QrCodeMessageTemplate.vb
'*******************************************************************************************'
'                                                                                           '
' Get API Key https://app.pdf.co/signup                                                     '
' API Documentation: https://developer.pdf.co/api/index.html                                '
'                                                                                           '
' Note: Replace placeholder values in the code with your API Key                            '
' and file paths (if applicable)                                                            '
'                                                                      '
'*******************************************************************************************'


Public Class QrCodeMessageTemplate

#Region "Constructors"

    Public Sub New()
    End Sub

    Public Sub New(ByVal Message As String)
        Me.Message = Message
    End Sub

#End Region


#Region "Properties"

    Public Property Message As String

#End Region

#Region "Overloaded Methods"

    Public Overrides Function ToString() As String

        If String.IsNullOrEmpty(Message) Then Return MyBase.ToString()

        Return Message

    End Function

#End Region


End Class

File: BarCode SDK/C#/QR Code with Message/QrCodeMessageTemplate.cs
//*******************************************************************************************//
//                                                                                           //
// Get Your API Key: https://app.pdf.co/signup                                               //
// API Documentation: https://developer.pdf.co/api/index.html                                //
//                                                                                           //
// Note: Replace placeholder values in the code with your API Key                            //
// and file paths (if applicable)                                                            //
//                                                                                           //
//*******************************************************************************************//


using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CreateBarCode
{
    class QrCodeMessageTemplate
    {
        #region Constructors

        public QrCodeMessageTemplate() { }
        public QrCodeMessageTemplate(string Url) { this.Message = Url; }

        #endregion

        #region Properties

        public string Message { get; set; }

        #endregion

        #region Overloaded Methods

        public override string ToString()
        {
            if(string.IsNullOrEmpty(Message))
            return base.ToString();

            return Message;
        }

        #endregion
    }
}
